tests/test_history.py

```diff
@@ -75,9 +75,9 @@ async def test_get_workflowtask_status(
             RESULTS[status].add(ID)
 
         # Create output_dataset and job
-        meta = dict(history=history)
+        history = history
```

Comment:
`history = history`? :)

Reply:
Just to be sure that history==history XD